middleware/jwtMiddleware.js

```diff
@@ -0,0 +1,18 @@
+const jwt = require('jsonwebtoken');
+const dotenv = require('dotenv');
+dotenv.config();
+
+const secret_phrase = process.env.SECRET_PHRASE;
```

Comment:
```suggestion
const secretPhrase = process.env.SECRET_PHRASE;
```

Be consistent with the case.